artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionBridge.java

```diff
@@ -127,9 +126,6 @@ public ClusterConnectionBridge(final ClusterConnection clusterConnection,
       this.managementNotificationAddress = managementNotificationAddress;
       this.flowRecord = flowRecord;
 
-      // we need to disable DLQ check on the clustered bridges
-      queue.setInternalQueue(true);
```

Comment:
Why did you change this?

It seems it doesn't have any connection to your change.

Beyond that.. we don't want any DLQ processing on internal queues from clustered bridge. This is a dangerous change.

Reply:
@clebertsuconic no it doesn't. But it's redundant because this has already been set during the creation of cluster connection bridge. See
org.apache.activemq.artemis.core.server.cluster.impl.ClusterConnectionImpl#nodeUP()

Reply:
Ok.. I see..

I have a small refactoring on this.. instead of trusting queue.isInternal I think we should move nodeUp to Bridge, and @override on the ClusterBridge.

I'm running some tests before I merge it.

Reply:
Thanks!